.mailmap

```diff
@@ -16,41 +16,83 @@
 #
 Abdón Rodríguez Davila <a@abdonrd.com> <a@abdonrd.com>
 Abigail J. Cross <across@us.ibm.com>
+Adrien Suau <adrien.suau@cerfacs.fr>
 Albert Frisch <albert.frisch@de.ibm.com>
 Albert Frisch <albert.frisch@de.ibm.com> <alfr@de.ibm.com>
 Ali Javadi-Abhari <ali.javadi@ibm.com> <ajavadia@users.noreply.github.com>
 Ali Javadi-Abhari <ali.javadi@ibm.com> <ajavadia@princeton.edu>
+Alejandro Pozas-Kerstjens <physics@alexpozas.com> <apozas@users.noreply.github.com>
+Almudena Carrera Vazquez <acv@zurich.ibm.com> <almudenacarreravazquez@hotmail.com>
 Andrew W. Cross <awcross@us.ibm.com>
 Andrew W. Cross <awcross@us.ibm.com> Andrew Cross <awcross@Andrews-MacBook-Pro.local>
 Andrew W. Cross <awcross@us.ibm.com> Andrew Cross <awcross@andrews-mbp.watson.ibm.com>
```

Comment:
any way to remove multiple duplicates? or is it unsafe?

Reply:
There isn't, they're committed to the git log and rewriting it would corrupt the history. That's what the mailmap file is for, adapting everything to a canonical form regardless of what is in the commit history. These duplicate lines just mean when you run the git log all the commits authored by Andrew show the same name and email address (he just happened to have commits authored with 3 email addresses).